docs/user-guide/docs-source/index.md

```diff
@@ -1,37 +1,13 @@
 # Welcome to AWS Ambit Scenario Designer for Unreal Engine 4
 
-AWS Ambit Scenario Designer for Unreal Engine 4 (Ambit) is a suite of tools to streamline content creation at scale for autonomous vehicle and robotics simulation applications.
+AWS Ambit Scenario Designer for Unreal Engine 4 (Ambit) is a suite of tools to streamline 3D content creation at scale for autonomous vehicle and robotics simulation applications. From procedural placement of obstacles to automatic generation of whole cities, Ambit can help you quickly create the 3D environments and scenarios you need for your simulations or other 3D applications.
```

Comment:
This is way outside of scope for this PR, but also consider adding that the plugin only works for Windows machines.

Reply:
Not outside the scope at all! Since these updates were focused on creating accurate setup documentation, calling out requirements like these is important. I've finessed the wording.